README.md

```diff
@@ -1,2 +1,74 @@
 # Awesome-Books
-This an app created using javascript
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This project is a book collection and shuffling application.
+
+## Description
+
+## Hosting
+
+This Portfolio is hosted on GitHub Page. [Click here](https://summyalena.github.io/Awesome-Books/)
+
+### Mobile version Screenshot:
+
+
+
+### Desktop version Screenshot:
+
+
```

Comment:
- [ ] I suggest cleaning up your documentation. If these sections are not in use, then kindly remove them to make a professional `README.md` file. 👌